TTHAnalysis/python/plotter/functions.cc

```diff
@@ -175,403 +185,59 @@ float DPhi_CMLep_Zboost(float l_pt, float l_eta, float l_phi, float l_M, float l
+float _puw2016_nTrueInt_EFG_upto279931[60] = {0.0012466094493921048, 0.013160935994534766, 0.01768708902983447, 0.021285749046576714, 0.02682365314703701, 0.020855099389744048, 0.020605061123741098, 0.016225941568981307, 0.014048132697170798, 0.02723810756281671, 0.08766748594327597, 0.2188586889136281, 0.3959226446275465, 0.53375637601322, 0.668298874408685, 0.7522202102811592, 0.7512257707056236, 0.8045384529297095, 0.790321346838139, 0.8694715549669896, 0.80985012101077, 0.8077764909536846, 0.9071025263393598, 1.0638089303658826, 1.17351814974311, 1.478754892106334, 1.5559650574410981, 1.8900984223481412, 1.9531327854179075, 2.2173732038373344, 2.402868150348858, 2.6038209490463284, 3.685234828496249, 4.657038942103952, 5.7091311002033045, 5.7091311002033045, 5.7091311002033045, 5.7091311002033045, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948, 1.0276435980365948};
+float puw2016_nTrueInt_EF(int nTrueInt) { if (nTrueInt<60) return _puw2016_nTrueInt_EF[nTrueInt]; else return 0; }
+float puw2016_nTrueInt_G_upto279931(int nTrueInt) { if (nTrueInt<60) return _puw2016_nTrueInt_G_upto279931[nTrueInt]; else return 0; }
+float puw2016_nTrueInt_EFG_upto279931(int nTrueInt) { if (nTrueInt<60) return _puw2016_nTrueInt_EFG_upto279931[nTrueInt]; else return 0; }
+
+
+TFile* puw2016_ICHEP = NULL;
+TFile* puw2016_ICHEP_Up = NULL;
+TFile* puw2016_ICHEP_Dn = NULL;
+TH1F* _puw2016_nInt_ICHEP = NULL;
+TH1F* _puw2016_nInt_ICHEP_Up = NULL;
+TH1F* _puw2016_nInt_ICHEP_Dn = NULL;
+float puw2016_nInt_ICHEP(float nInt, int var=0) { 
```

Comment:
@mmarionncern @folguera: can we move these to a new .cc file? (and maybe rename the global variables in a way that makes it obvious that 3 are tfiles and 3 are hists? )